WangscapeTest/TestMetaOutput.cpp

```diff
@@ -12,17 +12,18 @@
 #include <tilegen/TilesetGenerator.h>
 #include <tilegen/partition/TilePartitionerSquares.h>
 
-class TestMetaOutput : public ::testing::Test {
+#include "TestRequiringOptions.h"
+
+class TestMetaOutput : public TestRequiringOptions
+{
 protected:
-    std::string filename;
+    using TestRequiringOptions::TestRequiringOptions;
```

Comment:
This using clause and a succeding constructor call is more vague than it could be. That's true, there could be a fixture for tests using `Options`, but I think setting and reading an options filename could be taken out from this class. Note that it could be completely hidden:
```cpp
// OptionsFilename.h
std::string getOptionsFilename();
void setOptionsFilename(std::filename);

// OptionsFilename.cpp

namespace
{
std::string filename;
}

std::string getOptionsFilename()
{
    return filename;
}

void setOptionsFilename(std::string optionsFilename)
{
    filename = optionsFilename;
}
```

Then `setFilename(filename)` would be called in `main()` and `getFilename` would be used by `TestRequiringOptions`.

What do you think about it?